pandoc.cabal

```diff
@@ -283,8 +283,7 @@ Flag old-locale
 Library
   Build-Depends: base >= 4.7 && < 5,
                  syb >= 0.1 && < 0.8,
-                 split == 0.2.3.2,
-                 HUnit == 1.6.0.0,
+                 split >= 0.2 && <= 0.3,
```

Comment:
You only use one function from split, `splitOn`.
You could replace `splitOn ":"` with `splitBy (==':')`, using `splitBy` from `Text.Pandoc.Shared`.
Then `split` can be removed from cabal dependencies (generally, we should avoid adding new dependencies unless it's really needed).